apps/web-giddh/src/app/shared/template-froala/template-froala.component.ts

```
@@ -340,20 +343,51 @@ export class TemplateFroalaComponent implements OnInit {
-        this.emailForm.get(EmailType.Cc)?.patchValue(this.selectedCcEmails);
+    public onSubmit(type: string): void {
         if (this.emailForm.invalid) {
             return;
         }
-        let req = {
-            voucherType: this.voucherType,
-            model: this.emailForm.value
-        }
+
+        const formValue = cloneDeep(this.emailForm.value);
+        delete formValue?.voucherTypes;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)